ember-cli-build.js

```diff
@@ -52,44 +52,52 @@ module.exports = function (defaults) {
     'ember-simple-auth': {
       useSessionSetupMethod: true, //can be removed in ESA v5.x
     },
+    minifyCSS: {
```

Comment:
i don't think this is necessary anymore. i took this out and ran it with the `--environment=production` flag on `ember build/serve`, and it worked just fine. yanking embroider might be enough to set things right again.

Reply:
Without that we lose border color on the calendar and other random places where color just doesn't get applied. It actually matches production then as @dartajax noted above, but production is currently broken.

Reply:
i stand corrected :+1: